src/main/java/io/anserini/util/DumpAnalyzedQueries.java

```diff
@@ -48,12 +52,35 @@ public static class Args {
 
     @Option(name = "-output", metaVar = "[file]", required = true, usage = "queries")
     public String output;
+
+    @Option(name = "-language", usage = "Analyzer Language")
+    public String language = "en";
+  }
+
+  static Analyzer getAnalyzer(Args args) {
+    try {
+      if (AnalyzerMap.analyzerMap.containsKey(args.language)) {
+        System.out.println("Using language-specific analyzer");
```

Comment:
Can you swap these print statements for logs? See (https://github.com/castorini/anserini/blob/ec03f8cdf8bd9f51ce9f45233350cda32a6d23f6/src/main/java/io/anserini/analysis/AnalyzerMap.java#L28)